lib/features/preview-support/PreviewSupport.js

```diff
@@ -72,15 +61,9 @@ PreviewSupport.$inject = [
   'styles'
 ];
 
-PreviewSupport.prototype.cleanUp = function() {
-  var self = this;
-
-  forEach(self._clonedMarkers, function(clonedMarker) {
-    svgRemove(clonedMarker);
-  });
-
-  self._clonedMarkers = {};
-};
+// Markers are cleaned up with visuals, keep stub for compatibility
+// cf. https://github.com/camunda/camunda-modeler/issues/4307
+PreviewSupport.prototype.cleanUp = function() {};

```

Comment:
We could consider to log a warning here that this is no longer being used (and will be removed in future major releases). 

Cf. ContextPad.getPad or other instances.

Reply:
+1